src/lib/components/Feedback/index.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * @fileoverview Element that handles feedback on posts.
+ */
+import {html} from 'lit-element';
+import {BaseElement} from '../BaseElement';
+import {trackEvent} from '../../analytics';
+import './_styles.scss';
+
+/**
+ * Catches feedback submission on posts to submit to Analytics.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Feedback extends BaseElement {
+  constructor() {
+    super();
+    this.submitted = false;
+    this.submit = this.submit.bind(this);
+  }
+
+  submit(event) {
+    event.preventDefault();
+    const form = new FormData(event.target);
+    const formIterable = Array.from(form.entries());
+    if (!formIterable.length || this.submitted) {
+      return;
+    }
+
+    this.submitted = true;
+    this.requestUpdate();
+
+    formIterable.forEach((entry) => {
+      trackEvent({
+        category: 'Feedback',
+        action: 'submit',
+        label: entry[0],
+        value: Number(entry[1]),
+      });
+    });
+  }
+
+  render() {
+    // Because we share CSS with the `app.css` we want this to be in the light DOM
+    return html`
+      <details class="w-details">
+        <summary class="w-details__summary">
+          <h2 class="w-details__header">Give feedback</h2>
+        </summary>
+        <div
+          class="w-display--flex w-justify-content--center ${!this.submitted &&
+            'hidden'}"
+        >
+          <div class="w-subscribe--padded">
+            <p class="w-text--center w-mt--non">
```

Comment:
```suggestion
            <p class="w-feedback__confirmation w-text--center w-mt--non w-force-focus" tabindex="-1">
```

Adding `tabindex=-1` here will allow us to call `focus()` on this element after the user hits submit so a screen reader will announce the confirmation message. The `force-focus` class will suppress the focus outline so sighted users don't see a blue ring on the paragraph.